Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -399,6 +407,57 @@ theorem comp_mul_comm (T S : CentroidHom α) (a b : α) : (T ∘ S) (a * b) = (S
   rw [map_mul_right, map_mul_left, ← map_mul_right, ← map_mul_left]
 #align centroid_hom.comp_mul_comm CentroidHom.comp_mul_comm
 
+local notation "L" => AddMonoid.End.mulLeft
+local notation "R" => AddMonoid.End.mulRight
+
+/-- Ring hom from the centroid into the additive monoid endomorphisms. -/
+def toEndRingHom (α : Type*) [NonUnitalNonAssocSemiring α] :
```

Comment:
This result is also in #8742

Reply:
#8742 is on the queue, so this PR will need to merge master afterwards.

Reply:
This has now been done.